packages/site/pages/components/icon.js

```diff
@@ -41,7 +41,7 @@ const CommonSet = props => (
         const Comp = Icons[id]
         return (
           <div className="icon" key={i}>
-            <Comp size={sizes.large} className="svg" />
+            <Comp size={sizes.medium} className="svg" />
```

Comment:
```suggestion
            <Comp size={sizes.medium} />
```

Can also remove this (unused).

Reply:
@EdwardIrby I used the "Commit suggestion" button in Github to incorporate this feedback from Jake, but then it committed without a commit message and that's what's causing the build failure. Not sure how to fix that.